expressEngine/model/collection/order.js

```diff
@@ -20,6 +20,10 @@ const ordersSchema = new Schema({
     type: String,
     required: false,
   },
+  status: {
+    type: String,
+    required: true,
```

Comment:
Is required value need to be true in status field?? I think it should be false. If not, please clarify me.

Reply:
@gkgbunny every order have status, like order is canceled, pending, processing, etc.So i think it should be required true.

Reply:
@chiragSuccessive Then you must define the status type to Enum and specify the values in it

Reply:
@aman-successive Deepak is working on it.